contracts/l1/proxy/InitializableAdminUpgradeabilityProxyMock.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity 0.6.12;
```

Comment:
We could move this file into the `mocks` folder. The only goal of this file is that hardhat compiles the code, like other contracts in `mocks` folder.